raisinbread/instruments/bmi.linst

```diff
@@ -1,23 +1,24 @@
+table{@}bmi
 title{@}BMI Calculator
 date{@}Date_taken{@}Date of Administration{@}2006{@}2012
 static{@}Candidate_Age{@}Candidate Age (Months)
 static{@}Window_Difference{@}Window Difference (+/- Days)
 select{@}Examiner{@}Examiner{@}NULL=>''
 static{@}{@}Note: Please enter measurements in Imperial OR Metric units
-select{@}unit_classification{@}Choose unit of measurement{@}NULL=>''{-}'metric'=>'Metric'{-}'imperial'=>'Imperial'
+select{@}unit_classification{@}Choose unit of measurement{@}NULL=>''{-}'metric'=>'Metric'{-}'imperial'=>'Imperial'{-}'standard'=>'Standard'{-}'not_answered'=>'Not Answered'
 page{@}{@}Imperial units
 static{@}{@}Imperial units
-numeric{@}height_feet{@}Your height (feet) :
+numeric{@}height_feet{@}Your height (feet) :{@}undefined{@}undefined
```

Comment:
i think this 'undefined' might be coming from a bug in the instrument builder? unless this is the max/min in which case i think leaving it blank should be better. not sure if 'undefined' case is being handled in the NDB_BVL_Instrument_LINST class